near/omni-bridge/src/lib.rs

```diff
@@ -1262,8 +1273,14 @@ impl Contract {
             .sdk_expect("The transfer does not exist")
     }
 
-    pub fn is_transfer_finalised(&self, transfer_id: TransferId) -> bool {
-        self.finalised_transfers.contains(&transfer_id)
+    pub fn is_transfer_finalised(&self, transfer_id: &UnifiedTransferId) -> bool {
```

Comment:
This break the backward compatibility, let's add a new method, or accept two kind of arguments using option